tensorflow_addons/examples/Normalizations.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<p>It looks like this PR has two notebooks. Could you please remove one and move the more complete example to the directory mentioned above</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>